src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double firstSide;
+    private double secondSide;
+
+    public Rectangle(String color, double firstSide, double secondSide) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (double) Math.round((firstSide * secondSide) * 10) / 10;
```

Comment:
Avoid using magic numbers in your calculations. The number '10' used for rounding should be defined as a constant with a clear name that indicates its purpose. Moreover, consider if rounding is necessary here, as it might be better handled at the point of output if needed.